cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -46,6 +46,7 @@ def ensure_vm_running(small_vm):
         raise Exception("Unknown power state - unable to continue!")
 
 
+@pytest.mark.rhv1
```

Comment:
Definitely not sanity

Reply:
This is about changing memory/cpu of a VM. A basic feature added in cfme-5.8. We had many bugs in this area.